apps/lfx-one/src/app/modules/meetings/meetings-dashboard/meetings-dashboard.component.html

```
@@ -202,21 +206,26 @@ <h3 class="text-gray-600 mt-2">{{ activeLens() === 'me' ? 'You have no upcoming
+            @if (timeFilter() === 'pending-rsvp') {
+              <h3 class="text-xl font-semibold text-gray-900 mt-4">You're all caught up</h3>
+              <p class="text-gray-600 mt-2">No pending RSVPs — all your invites have a response.</p>
+            } @else {
+              <h3 class="text-xl font-semibold text-gray-900 mt-4">No Meetings Found</h3>
+              <p class="text-gray-600 mt-2">Try adjusting your search or filter criteria</p>
+            }
           </div>
         </div>
       </lfx-card>
-    } @else if (filteredMeetings().length > 0) {
+    } @else if (visibleMeetings().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)